pkg/services/oracle/jsonpath/jsonpath.go

```diff
@@ -219,6 +228,9 @@ func (p *pathParser) descendRecursive(objs []interface{}) ([]interface{}, bool)
 	for len(objs) > 0 {
 		newObjs, _ := p.descendByIdentAux(objs, false, val)
 		values = append(values, newObjs...)
+		if maxObjects < len(values) {
```

Comment:
Some checks are post-append, some pre-append, maybe it's better to have more uniform style?

Reply:
Fixed.